digid_eherkenning/types.py

```diff
@@ -93,5 +98,5 @@ class EHerkenningSAMLConfig(TypedDict):
     sp: ServiceProviderSAMLConfig
     idp: IdentityProviderSAMLConfig
     security: SecuritySAMLConfig
-    contactPerson: dict
+    contactPerson: ContactPerson
```

Comment:
Isn't this a dict with keys `technical` and `administrative` which are then of type `ContactPerson`?

Reply:
eh yeah, I was very confusing when looking at this type definition and where it was used. I'm afraid we're still a long way removed from making this package type safe :(